extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+class WikiRecommendations {
+
+	const THUMBNAIL_WIDTH = 100;
+	const THUMBNAIL_RATIO = 16 / 9;
+
+	const RECOMMENDATIONS = [
+		'en' => [
+			[
+				'thumbnailUrl' => 'https://vignette1.wikia.nocookie.net/gameofthrones/images/3/3a/WhiteWalker_%28Hardhome%29.jpg/revision/latest?cb=20150601151110',
+				'title' => 'Game of Thrones',
+				'url' => 'http://gameofthrones.wikia.com/wiki/Game_of_Thrones_Wiki',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette2.wikia.nocookie.net/deathnote/images/1/1d/Light_Holding_Death_Note.png/revision/latest?cb=20120525180447',
+				'title' => 'Death Note',
+				'url' => 'http://deathnote.wikia.com/wiki/Main_Page',
+			],
+			[
+				'thumbnailUrl' => 'https://vignette2.wikia.nocookie.net/midnight-texas/images/b/b0/Blinded_by_the_Light_106-01-Rev-Sheehan-Davy-Deputy.jpg/revision/latest?cb=20170820185915',
+				'title' => 'Midnight Texas',
+				'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
+			],
+		],
+		'de' => [],
+		'fr' => [],
+		'es' => [],
+		'pt-br' => [],
+		'ru' => [],
+		'it' => [],
+		'pl' => [],
+		'zh' => [],
+		'ja' => [],
+	];
+
+	const DEV_RECOMMENDATIONS = [
```

Comment:
why this array has different structure than `RECOMENDATIONS`? (language as a key is there)